tests/ci_build/Dockerfile.cpu

```diff
@@ -28,7 +28,17 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 
 # Create new Conda environment
 COPY conda_env/cpu_test.yml /scripts/
-RUN mamba env create -n cpu_test --file=/scripts/cpu_test.yml
+RUN mamba env create -n cpu_test --file=/scripts/cpu_test.yml && \
+    mamba create -y -n build_env -c conda-forge python=3.9
```

Comment:
is 3.9 needed? The XGBoost package claims support for 3.8 and most (all?) other tests are on 3.8.